lib/errors/error_handling.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/errors/course_update_error_handling"
+
+module ErrorHandling
+  include CourseUpdateErrorHandling
+
+  def perform_error_handling(error, typical_errors, extra, course, optional_params)
```

Comment:
This is probalby going to be easier to keep track of with keyword args.

Reply:
Done!